installation/proc-install-rhdh-operator.adoc

```diff
@@ -0,0 +1,49 @@
+[id='proc-install-rhdh-operator_{context}']
+= Installing {product} using the Operator
```

Comment:
I notice we have helm chart installation instructions in BOTH the admin guide and getting started guide (because it can be done by an admin or a user); however this new doc for installing via operator is ONLY in the getting started guide, which is wrong since you need to be cluster admin to install via operator. 

ONCE the operator is installed, THEN a user can enable it via the `Switch to *Developer* mode on your Red Hat OpenShift web console` step and beyond.

So we need instuctions for enabling the operator via admin guide, then doc for enabling it as a user in the getting starting guide, with a link back to the admin guide for the prereq steps).

Reply:
@nickboldt Should we also include the full procedure for enabling the operator in the Admin guide i.e. as we have them in the User guide (without the admin steps)? Please see latest previews of both guides.

Reply:
seems redundant -- why not just cross-link the sections?